apps/src/pcd_organized_multi_plane_segmentation.cpp

```diff
@@ -194,11 +196,15 @@ class PCDOrganizedMultiPlaneSegmentation {
                 << std::endl;
       typename pcl::PointCloud<PointT>::ConstPtr approx_contour_const = approx_contour;
 
-      for (std::size_t idx = 0; idx < approx_contour->points.size(); ++idx) {
-        sprintf(name, "approx_plane_%02d_%03d", int(i), int(idx));
+      for (std::size_t idx = 0; idx < approx_contour->size(); ++idx) {
+        std::snprintf(name,
+                      sizeof(name),
+                      "approx_plane_%02zu_%03zu",
+                      static_cast<std::size_t>(i),
+                      static_cast<std::size_t>(idx));
```

Comment:
Just to clarify my policy. When we're printing things coming from `size()` method, then the explicit casting is necessary, when printing actual indexes from the for loop, I would leave those changes to take place once the actual index type is changed and not right now.

Reply:
`idx` and `i` are already `size_t`. Will keep this in mind going forwards

Reply:
Should we not remove the casts?